name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/preferences/BackupsPreferencePage.java

```diff
@@ -0,0 +1,128 @@
+
+        String relativeEditorMessage = MessageFormat.format("Relativer Backup Order (z.B. {0})",
+                        Paths.get(".", "back").toString()); //$NON-NLS-1$ //$NON-NLS-2$
+        backupFolderRelativeEditor = new StringFieldEditor(UIConstants.Preferences.BACKUP_FOLDER_RELATIVE,
+                        relativeEditorMessage, getFieldEditorParent());
+
+        addField(backupModeEditor);
+        addField(backupFolderAbsoluteEditor);
+        addField(backupFolderRelativeEditor);
+
+        createRelativeFolderInfo(getFieldEditorParent());
+    }
+
+    protected void createRelativeFolderInfo(Composite composite)
+    {
+        relativeFolderInfoLabel = new Label(composite, SWT.WRAP);
+        relativeFolderInfoLabel.setFont(getFieldEditorParent().getFont());
+
+        GridDataFactory.fillDefaults().span(2, 1).grab(true, true).applyTo(relativeFolderInfoLabel);
+    }
+
+    @Override
+    protected void initialize()
+    {
+        super.initialize();
+        setBackupFolderEnabled(backupModeEditor.getBooleanValue());
+
+        if (backupModeEditor.getBooleanValue())
+        {
+            onRelativeBackupFolderChange(backupFolderRelativeEditor.getStringValue());
+        }
+        else
+        {
+            onRelativeBackupFolderChange(null);
+        }
+    }
+
+    @Override
+    public void propertyChange(PropertyChangeEvent event)
+    {
+        super.propertyChange(event);
+
+        if (event.getProperty().equals(FieldEditor.VALUE) && event.getSource().equals(backupModeEditor))
+        {
+            setBackupFolderEnabled((Boolean) event.getNewValue());
+            onRelativeBackupFolderChange(backupFolderRelativeEditor.getStringValue());
+        }
+        else if (event.getProperty().equals(FieldEditor.VALUE) && event.getSource().equals(backupFolderRelativeEditor))
+        {
+            onRelativeBackupFolderChange((String) event.getNewValue());
+        }
+    }
+
+    private void setBackupFolderEnabled(boolean enabled)
+    {
+        backupFolderAbsoluteEditor.setEnabled(!enabled, getFieldEditorParent());
+        backupFolderRelativeEditor.setEnabled(enabled, getFieldEditorParent());
+        relativeFolderInfoLabel.setEnabled(enabled);
+    }
+
+    private void onRelativeBackupFolderChange(String newValue)
+    {
+        if (null == newValue || newValue.trim().equals(""))//$NON-NLS-1$
+        {
+            relativeFolderInfoLabel.setText("");//$NON-NLS-1$
+            return;
+        }
+
+        Path currentDataFile = Path.of("C:\\temp\\bla.xml");
+        Path currentDataFolder = currentDataFile.getParent();
+        Path dest = currentDataFolder.resolve(newValue).normalize();
+
+        String infoText = MessageFormat.format(
+                        "Backups für die aktuelle Datendatei {1} werden in den folgenden Ordner gespeichert: {0}", dest,
+                        currentDataFile);
+        relativeFolderInfoLabel.setText(infoText);

```

Comment:
Please remove the windows specific path here.

In the preferences, we do not have access to a specific file. Therefore, I believe you can only print the name of the sub-folder, e.g. "backups".

I believe in a separate change, we should add the information where backups are written - somewhere to the file. At the moment, there is no good place where to do this. Maybe we extend the tooltip with the name of the file (currently shows the full path of the file). Maybe we add a tab to the "settings" page where bookmarks and attributes are configured. Anyway, let's keep that to a separate pull request (this one is about the backup mode).